gradle.properties

```diff
@@ -8,7 +8,7 @@ pluginVersion = 1.0.1
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html#intellij-platform-based-products-of-recent-ide-versions
 # for insight into build numbers and IntelliJ Platform versions.
 pluginSinceBuild = 202
-pluginUntilBuild = 203.*
+pluginUntilBuild = 222.*
```

Comment:
temporary change version to be able to run on Android Studio Flamingo | 2022.2.1. (Build #AI-222.4459.24.2221.9862592)